print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
```

Comment:
Should we add context here to explain what this Node class is supposed to do? Looks like a linked list to me, but some context might be helpful for others!